### DISCUSSION TITLE
翻译错误

### DISCUSSION THREAD

**Discussion by AloneWeekend** [immersive-translate/immersive-translate#1890] [Help]
### 概述 | Summary

I remember freaking out when I hit 300 subscribers 11 years ago.. lol翻译成了我记得 300 年前我达到 11 名订阅者时吓坏了......哈哈
![屏幕截图 2024-07-12 165544](https://github.com/user-attachments/assets/d1a2e247-6200-4489-9fcb-91b3fd3026ac)

**Comment by theowenyoung** [comment]
什么翻译服务
